07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    ).columns[1:].values
</pre><p>Just to keep in mind, the new version of <code>pandas</code> has deprecated <code>values</code>, and <code>to_numpy()</code> should be used instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>This is good to know! I didn't know that.</p><p> </p><p>I'll fix it for this script, and I'll create an issue to remind myself about it going forward. I'm not planning to update my version of <code>pandas</code> imminently but if I do in the future it may be good to remember why things are breaking, haha. </p>